.gitignore

```diff
@@ -63,7 +63,20 @@ kratos/includes/kratos_version.h
 applications/FluidDynamicsApplication/test_examples/fs_cavity/ProjectParameters\.py
 
 applications/FluidDynamicsApplication/test_examples/fs_cavity/run_test\.py
-
+applications/ULFapplication/test_examples/TRIALS/
```

Comment:
You can not modify gitignore, please use the .git/info/exclude instead-> is personal